src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
```

Comment:
you renamed not all your variables

```suggestion
  const calculator = {
```